gui/src/qmlmainwindow.cpp

```diff
@@ -55,6 +55,13 @@ static QString shader_cache_path()
     return path;
 }
 
+
+static const char *render_params_path()
+{
+    static QString path = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) + "/Chiaki/pl_render_params.conf";
```

Comment:
I'm not really sure about this one, it did correctly resolve to `~/.config/Chiaki` on Linux for me, but the Qt6 docs say that on Windows it resolves to `"C:/Users/<USER>/AppData/Local/<APPNAME>"`, i..e  the added `/Chiaki/` would not be neccessary in this case.
I don't remember why I didn't use `QStandardPaths::AppConfigLocation` 🤔

Reply:
Yeah this should be `QStandardPaths::AppConfigLocation`